micrometer-core/build.gradle

```diff
@@ -143,13 +143,15 @@ dependencies {
 
     // dependency injection tests
     testImplementation 'javax.inject:javax.inject'
-    testImplementation 'org.springframework:spring-context'
     testImplementation 'com.google.inject:guice'
 
     // Uncomment these if you are interested in testing injection with dagger in MeterRegistryInjectionTest
     // testImplementation 'com.google.dagger:dagger'
     // testAnnotationProcessor 'com.google.dagger:dagger-compiler'
 
+    // Only needed for ModifiedClassPathClassLoader
+    testImplementation(libs.spring5.core)
```

Comment:
I'd like to get rid of this test dependency as well, but that's a more involved change than I want to make now to solve the issue of test failures on bumping the Spring version.